src/ApiCalls/postMethodRequest.js

```diff
@@ -35,6 +35,22 @@ const loginHandler = async (userInput, authDispatch, navigate) => {
     }
 }; 
 
+const postTestLoginUser = async(authDispatch, navigate) => {
```

Comment:
Suggestion: you don't require two separate functions for login just pass the guest login credentials in liginHandler itself